### PR TITLE
updated tutorials

### PR DESCRIPTION
**Pull request by jyeatman**

### CODE REVIEW

File: docs/bibliography/diffusion-mri.md [hunk 1]
```diff
@@ -1,18 +1,32 @@
 # Resources for learning the basics of diffusion MRI
 
-## Here as some basic readings to introduuce the field
+## Here are some readings to introduuce the field
```
Comment (arokem) [inline docs/bibliography/diffusion-mri.md]:
```suggestion
## Here are some readings to introduce the field
```

File: docs/bibliography/diffusion-mri.md [hunk 2]
```diff
@@ -1,18 +1,32 @@
...
 
-Wandell BA (2016) Clarifying Human White Matter. Annu Rev Neurosci 39(1):annurev-neuro-070815-013815.
+**A great overview of the field:**
 
-## Automated Fiber Quantification is a method for analyzing white matter tracts
+    Wandell BA (2016) Clarifying Human White Matter. Annu Rev Neurosci 39(1):annurev-neuro-070815-013815.
```
Comment (arokem) [inline docs/bibliography/diffusion-mri.md]: When these are indented, the look like  code blocks  (see here: https://deploy-preview-28--afqdocs.netlify.app/bibliography/diffusion-mri/):

Maybe use block-quotes for these instead?
```suggestion
> Wandell BA (2016) Clarifying Human White Matter. Annu Rev Neurosci 39(1):annurev-neuro-070815-013815.
```

File: docs/bibliography/diffusion-mri.md [hunk 3]
```diff
@@ -1,18 +1,32 @@
...
 
-Here is the paper that introducued the method:
+**Review paper on fiber tractography methods:**
+
+    Jeurissen B, Descoteaux M, Mori S, Leemans A (2019) Diffusion MRI fiber tractography of the brain. NMR Biomed 32(4):1–22.
+
+**Two papers from our group focused on using diffusion MRI to study brain development and plasticity:**
+
+    Huber E, Donnelly PMPM, Rokem A, Yeatman JD (2018) Rapid and widespread white matter plasticity during an intensive reading intervention. Nat Commun 9(1):2260.
```
Comment (arokem) [inline docs/bibliography/diffusion-mri.md]: Does Patrick really have four initials?

**Review by arokem — COMMENTED** [review]
Thanks! Here are some comments/suggestions

File: docs/bibliography/diffusion-mri.md (hunk 3 above)
Comment (jyeatman) [reply docs/bibliography/diffusion-mri.md]: No but I can't figure out how to resolve this through the GUI

Comment (jyeatman) [reply docs/bibliography/diffusion-mri.md]: It should be Donnelly PM

Comment (arokem) [reply docs/bibliography/diffusion-mri.md]: OK - I think that I just resolved that one.

File: docs/bibliography/diffusion-mri.md [hunk 4]
```diff
@@ -1,18 +1,32 @@
...
-Wandell BA (2016) Clarifying Human White Matter. Annu Rev Neurosci 39(1):annurev-neuro-070815-013815.
+**A great overview of the field:**
 
-## Automated Fiber Quantification is a method for analyzing white matter tracts
+> Wandell BA (2016) Clarifying Human White Matter. Annu Rev Neurosci 39(1):annurev-neuro-070815-013815.
 
-Here is the paper that introducued the method:
+**Review paper on fiber tractography methods:**
+
+    Jeurissen B, Descoteaux M, Mori S, Leemans A (2019) Diffusion MRI fiber tractography of the brain. NMR Biomed 32(4):1–22.
```
Comment (arokem) [inline docs/bibliography/diffusion-mri.md]:
```suggestion
> Jeurissen B, Descoteaux M, Mori S, Leemans A (2019) Diffusion MRI fiber tractography of the brain. NMR Biomed 32(4):1–22.
```

File: docs/bibliography/diffusion-mri.md [hunk 5]
```diff
@@ -1,18 +1,32 @@
...
+
+**Two papers from our group focused on using diffusion MRI to study brain development and plasticity:**
+
+    Huber E, Donnelly PM, Rokem A, Yeatman JD (2018) Rapid and widespread white matter plasticity during an intensive reading intervention. Nat Commun 9(1):2260.
+
+    Huber E, Henriques RN, Owen JP, Rokem A, Yeatman JD (2019) Applying microstructural models to understand the role of white matter in cognitive development. Dev Cogn Neurosci 36(February):100624.
```
Comment (arokem) [inline docs/bibliography/diffusion-mri.md]:
```suggestion
> Huber E, Donnelly PM, Rokem A, Yeatman JD (2018) Rapid and widespread white matter plasticity during an intensive reading intervention. Nat Commun 9(1):2260.

> Huber E, Henriques RN, Owen JP, Rokem A, Yeatman JD (2019) Applying microstructural models to understand the role of white matter in cognitive development. Dev Cogn Neurosci 36(February):100624.
```

File: docs/bibliography/diffusion-mri.md [hunk 6]
```diff
@@ -1,18 +1,32 @@
...
+Tractometry refers to methods for analzying properties of the brain's major white matter connections. Automated Fiber Quantification (AFQ) is a tractometry toolbox we developed for analyzing white matter tracts in individuals and groups. There is a MATLAB and a Python version of AFQ.
+
+**Here is the paper that introducued the method:**
 
     Yeatman JD, Dougherty RF, Myall NJ, Wandell BA, Feldman HMM (2012) Tract profiles of white matter properties: automating fiber-tract quantification. PLoS One 7(11):e49790.
 
 And here is a more recent paper introducing a browser-based visualization and analysis tool to support reproducibility and data sharing
 
-    Yeatman JD, Richie-Halford A, Smith JKJK, Keshavan A, Rokem A (2018) A browser-based tool for visualization and analysis of diffusion MRI data. Nat Commun 9(1):940.
+    Yeatman JD, Richie-Halford A, Smith JK, Keshavan A, Rokem A (2018) A browser-based tool for visualization and analysis of diffusion MRI data. Nat Commun 9(1):940.
```
Comment (arokem) [inline docs/bibliography/diffusion-mri.md]:
```suggestion
> Yeatman JD, Richie-Halford A, Smith JK, Keshavan A, Rokem A (2018) A browser-based tool for visualization and analysis of diffusion MRI data. Nat Commun 9(1):940.
```

**Review by arokem — COMMENTED** [review]
Maybe change all the cites to use block-quotes? Ideally, we would also put these in the bib file (https://github.com/nrdg/afq-docs/blob/master/bibliography.bib), so that it renders and links nicely.

**Comment by arokem** [comment]
Let's go! Thanks!

**Pull request merged**